pkey/gcs/gcs.go

```diff
@@ -50,4 +50,56 @@ func (g *GCS) GetPrivateKeyData(ctx context.Context, appId int64) ([]byte, error
 	return ioutil.ReadAll(r)
 }
 
-func (g *GCS) Close() error { return g.cli.Close() }
+func (p *Provider) Close() error { return p.cli.Close() }
+
+type Processor struct {
+	processor pkey.Processor
```

Comment:
It shouldn't be here. If you want to write stuff somewhere _and_ to GCS at the same time, implement a `MultiProcessor`.

Reply:
Done.